### PR TITLE
PyTests for Relationship Server

### PR DESCRIPTION
**Pull request by meghanabhange**
Tests for 
- Nationality `Indian Student travelling from Wuhan`
- Relationships `Son of P3`
- Travel `Traveled from Italy`
- Travel with acronyms `Traveled from UK`
- Travel with aliases `Traveled from Republic of South Korea`

### CODE REVIEW

File: test_relationship_server.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
+import pytest
```
Comment (NirantK) [inline test_relationship_server.py:1]: How can we run this test without commenting `app.run`?

File: test_relationship_server.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+from relationship_server import record_processor
+
+
+def test_record_processor():
```
Comment (NirantK) [inline test_relationship_server.py:6]: Add tests for missing keys with a patient id.

**Review by NirantK — CHANGES_REQUESTED** [review]

File: test_relationship_server.py (hunk 1 above)
Comment (NirantK) [reply test_relationship_server.py:1]: Resolved as per discussion on Telegram. Deciding to use the commented `app.run` option. 

Based on @meghanabhange's information that PythonAnywhere makes us comment that anyway.

File: test_relationship_server.py (hunk 2 above)
Comment (meghanabhange) [reply test_relationship_server.py:6]: solved by 78cbd9b

**Pull request merged**